lib/sorbet-rails/config.rb

```diff
@@ -54,6 +54,7 @@ def initialize
         :active_record_named_scope,
         :active_record_querying,
         :active_relation_where_not,
+        :active_record_serialization,
```

Comment:
let's rename this to be clearer: `active_record_serialized_attribute`

Reply:
Nice.  Makes sense to me.  I've renamed the plugin.